platforms/org.eclipse.mita.platform.arduinouno/xtend-gen/org/eclipse/mita/platform/arduinouno/connectivity/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.LedGenerator.java._trace
```

Comment:
Please add `xtend-gen/**` to your `.gitignore` rather than explicitly listing all the files.

Reply:
👍